index.html

```diff
@@ -0,0 +1,164 @@
+        <img class="toggle" src="images/toggle.png" alt="toggle-bar">
+        <ul class="first-header">
+            <li class="top-menu"><img class="facebook" src="images/facebook-icon.jpg" alt="facebook icon"></li>
+            <li class="top-menu"><img class="twitter" src="images/twitter-icon.png" alt="facebook icon"></li>
+            <li class="top-menu">English</li>
+            <li class="top-menu">My Page</li>
+            <li class="top-menu">Layout</li>
+        </ul>
+        <nav class="second-nav hide-menu">
+            
+            <div class="logo__ul">
+                <div class="close">&times;</div>
+                <p class="logo">Logo</p>
+                <ul class="displayf">
+                    <li>About</li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                </ul>
+            </div>
+            <div class="campcc">
+                <p class="camp">cc Campaign</p>
+            </div>
+        </nav>
+        <section class="sections">
+            <section class="intro-section">
+                <p>"Hello! Sharing world"</p>
+                <h2>BUILDING A BLACKGIRLS TECH COMMUNITY 2022</h2>
+                <div class="box">
+                    <p>Creating a tech community to empower young black women to surface the tech field. Black girls stand.
+                    </p>    
+                </div>
+                <div class="date">
+                    <h3>2022.01.03(MON) ~ 8(SAT)</h3>
+                    <p>@ S.S.A.M.B empowerment of West and Central Africa</p>
+                </div>
+            </section>
+            <section class="section-2">
+                <p class="main-p">Main Program</p>
+                <hr>
+                <div class="cards">
+                    <div class="card1">
+                        <div class="index">
+                        <img class="fas" src="images/icons8-picture-64.png" alt="picture description">
+                        </div>
+                    <h5>About</h5>
+                    <P>SSAMB is made up of 5 talented and career driven youths whose aim is to empower young black girls into the tech industry</P>
+                    </div><br>
+                    <div class="card1">
+                        <img class="fas" src="images/icons8-picture-64.png" alt="image description">
+                        <h5>Eligible</h5>
+                        <p>All black girls aspiring to get into the field are highly encourage to take advantage of this opportunity.</p>
+                    </div><br>
+                    <div class="card1">
+                        <img class="fas" src="images/icons8-box-64.png" alt="images">
+                        <h5>Career</h5>
+                        <p>The community makes it flexible for you to pick out any tech field of your. we train you from zero to hero.</p>
+                    </div><br>
```

Comment:
- [ ] You have done a great job with these cards, but when I look keenly at them their content is not well centered. You should utilize aligning them with flexbox.

Reply:
RESOLVED.
Cards aligned properly using flexbox.